src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -983,6 +1003,12 @@ def update_connected_cluster(cmd, client, resource_group_name, cluster_name, htt
 
     release_namespace = validate_release_namespace(client, cluster_name, resource_group_name, kube_config, kube_context, helm_client_location)
 
+    helm_values = get_all_helm_values(release_namespace, kube_config, kube_context, helm_client_location)
+
+    if helm_values.get('global').get('isLeastPrivilegesMode') is True:
+        if auto_upgrade is True:
+            raise InvalidArgumentValueError("Your cluster is running in least privileges mode. Autoupdates are not supported in this mode")
```

Comment:
collect exception telemetry for this

Reply:
is the same check being done during onboarding?

Reply:
We're setting auto-upgrade to false by default whenever it is in least privileges mode, so we don't need a check here, right? And there's no enable-auto-upgrade flag in connect command anyway